Sources/ArcGISToolkit/Resources/en.lproj/Localizable.strings

```diff
@@ -1,8 +1,20 @@
+/* No comment provided by engineer. */
+"(%lld)" = "(%lld)";
+
+/* No comment provided by engineer. */
+"%@" = "%@";
+
+/* No comment provided by engineer. */
+"%@ %@" = "%@ %@";
+
+/* No comment provided by engineer. */
+"%lf" = "%lf";
```

Comment:
Where is these used? Shouldn't there some text with these parameters?

Reply:
These are do to how certain `Text` views are created. A search for `"\(` finds (almost?) all of them. For example, "%@" is the result of `Text` being initialized like so: `Text("\(...)")`. That is done both in `AttachmentList` and `AttachmentPreview` to display the attachment size:

```swift
Text("\(attachmentModel.attachment.size.formatted(.byteCount(style: .file)))")
```

Instead, `Text.init(_:format:)` should be used instead:

```swift
Text(Int64(attachmentModel.attachment.size), format: .byteCount(style: .file))
```

If a string used to create a `Text` view should not be localized, use `Text.init(verbatim:)`.

Reply:
@dfeinzimer I just saw your commit. Here, nothing to localize. The parameters are returned as it is so you should use `Text.init(verbatim:)` for these cases.

Reply:
@njarecha I've gotten it down to two. I'm not sure where they may be coming from.

Reply:
One of the remaining ones is coming from this:

```swift
Text((item.result as? Double).map { "\($0)" } ?? "N/A")
```

It probably would be best to use an if statement there:

```swift
if let result = item.result as? Double {
    Text(result, format: ...)
} else {
    Text("N/A")
}
```

The other one must be coming from here somehow:

```swift
Text(
    "\(viewModel.pendingTrace.startingPoints.count, specifier: "%lld") selected",
    bundle: .toolkitModule,
    comment: "A label declaring the number of starting points selected for a utility network trace."
)
```

For that, the specifier shouldn't be necessary. See one of Apple's recent WWDC sessions on localization for how to localize plural strings.

Reply:
Thanks Phil, that took care of the first one. Removing the the specifier didn't seem to make any difference for the second. I poked around some more but haven't been able to find the source.

Reply:
Found it. The last one is coming from the title of a disclosure group (used in two places:

```swift
DisclosureGroup("(\(viewModel.selectedTrace?.elementResults.count ?? 0))", ...)
...
DisclosureGroup("(\(viewModel.selectedTrace?.utilityFunctionTraceResult?.functionOutputs.count ?? 0))", ...)
```

The solution is to use `DisclosureGroup.init(isExapnded:content:label:)` and in the label closure, return an instance of `Text` using `Text(_:format:)`.

Reply:
💰 like a treasure hunt!

Reply:
Yes, that was it, thank you again Phil!